ubiquity-fundraising-research-april-may-2024.md

```diff
@@ -0,0 +1,87 @@
+
+To obtain objective feedback and information, the conversations were conducted anonymously, in the format of interviews with experts, not as a pitch to investors. Experts were provided with Ubiquity's public materials and no internal information was provided. Some experts conducted their own research to comprehensively assess Ubiquity.
+
+The trigger for starting the research was Sergey's interaction in March-April 2024 with an agency that supported ChainGPT. Previously, Sergey believed that he objectively assessed Ubiquity's situation, but it turned out during the interaction with the agency that this might not be the case. According to the agency's assessment in March 2024, Ubiquity had low market potential.
+
+Sergey presumed that investors might come to a similar assessment, which would significantly hinder Ubiquity's ability to obtain the next funding round. Seeing Ubiquity through the eyes of independent expert investors, Alex and Sergey could build investor communication in such a way as to highlight Ubiquity's strengths and mitigate its weaknesses. Observing Alex's approach to finding investors, Sergey believes that this research will significantly save time for Ubiquity in preparing for the next funding round.
+
+This is a brief version of the report that represents information obtained from experts, aggregated by the end of June 2024. Part of the information and recommendations received from experts have not yet been aggregated and are not included in this report. The research took over 2 months almost full-time, while Sergey was only paid for one month. The rest of the information will be aggregated and provided later if Sergey has time or if he is paid for it.
+
+## Investments
+Ubiquity's ability to retain the amount of investments received in the previous round commands deep respect. Such a situation is a strong positive signal to investors that their funds are managed efficiently. It is desirable for such information to be confirmed by facts that investors can verify.
+
+### Funding Round
+Since Ubiquity has funds from the previous round, it remains unclear for what purposes to attract a new investment round. To justify the need for additional funding, it is necessary to provide a reasoned development strategy for Ubiquity with directions for using capital.
+
+Typically, more effective times for investments are autumn and spring, summer being the least active period. It is better to start contacting investors at the beginning of autumn 2024.
+
+### Investors
+Experts asked whether Ubiquity is approaching current investors with an offer to participate in a new funding round. If investors familiar with Ubiquity are expanding their investments, this is a positive signal. If none of the existing investors wish to expand their investments in Ubiquity, this could be a negative signal. Investors communicate with each other and will try to get feedback about Ubiquity from current investors.
```

Comment:
Our existing investors didn't provide any value even when tasked with simple requests like making an introduction. So I am not keen to provide very special treatment to the existing investors. However things are always open for negotiation.